packages/pwa-kit-react-sdk/src/webpack/plugins.js

```diff
@@ -96,6 +96,8 @@ export class BuildMarkerPlugin {
 }
 
 /**
+ * @deprecated will be removed in v2.0.0
+ *
```

Comment:
Did this really get released? It never actually validated anything. I appreciate taking things so seriously, but do we _really_ need to go through a deprecation process?

(Btw. this is why I was so vocal about not merging a no-op validator previously!)

Reply:
It was released in v1.4.0 and technically speaking it is exported as a public class... To be honest, I wasn't too sure if we really need to go thru this deprecation process either.